modules/nft/src/lib.rs

```diff
@@ -395,7 +395,7 @@ impl<T: Config> NFT<T::AccountId> for Pallet<T> {
 	type Balance = NFTBalance;
 
 	fn balance(who: &T::AccountId) -> Self::Balance {
-		orml_nft::TokensByOwner::<T>::iter_prefix(who).count() as u128
+		orml_nft::TokensByOwner::<T>::iter_prefix((who,)).count() as u128
```

Comment:
so what is the benifit of nmap? or could we just use child trie instead?

Reply:
Any prefix key can be iterated
```
orml_nft::TokensByOwner::<Runtime>::iter_prefix((AccountId,))
orml_nft::TokensByOwner::<Runtime>::iter_prefix((AccountId, ClassId))
```

Reply:
ok, my understanding is that no matter child trie or double map 2 levels  interation at most while with nmap  no limitation at all